andela/mervels-bn-frontend
# #168131516 Set up tests on the client

#### What does this PR do?
Sets up tests for the project
#### Description of Task to be completed?
Write a sample test to make sure tests are working.
#### How should this be manually tested?
- Clone this repo
- Checkout to this branch
- Run `npm test`
(You should see an output similar to the one in screenshots)
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#168131516](https://www.pivotaltracker.com/story/show/168131516)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/50402526/68203274-1c05d180-ffce-11e9-8f0e-2d04ff07db1d.png)

#### Questions:
N/A
